kolibri/utils/options.py

```diff
@@ -32,6 +32,28 @@
             "envvars": ("KOLIBRI_DATABASE_HOST",),
         },
     },
+    "Server": {
+        "THREAD_POOL": {
+            "type": "integer",
+            "default": 10,
+            "envvars": ("CHERRYPY_THREAD_POOL",),
+        },
+        "SOCKET_TIMEOUT": {
+            "type": "integer",
+            "default": 10,
+            "envvars": ("CHERRYPY_SOCKET_TIMEOUT",),
+        },
+        "QUEUE_SIZE": {
+            "type": "integer",
+            "default": 5,
+            "envvars": ("CHERRYPY_QUEUE_SIZE",),
+        },
+        "QUEUE_TIMEOUT": {
+            "type": "float",
+            "default": 0.1,
+            "envvars": ("CHERRYPY_QUEUE_TIMEOUT",),
```

Comment:
This naming seems to violate [our convention](https://docs.google.com/document/d/1v-_B7dFlEw9EAOfMSU78IkfhMX9DihzN7jjjxzEUfBk/edit#heading=h.otycu89b2guh) that all env vars should have the `KOLIBRI_` prefix.

At a higher level, is there a strong need to make these configurable? I think we should be careful about expanding the scope of the settings too quickly.

(cc @jamalex)

Reply:
I was not aware of that convention. I will change it.

About the other question: we talked to make them configurable so we can test the different options in the partners' installations. I think it's worth to have them. The defaults are the optimized parameters we have tuned in our tests but it could happen in other environments with more/less users, better/worse bandwidth, equipement, etc. the parameter should be changed.

Feel free to close this PR if you consider having these options is a bad idea. In such case I will do a new PR to set the best defaults we can tune in kolibri.